lighthouse/__init__.py

```diff
@@ -57,3 +57,9 @@ def setup_routes(app):
 
         app.register_blueprint(v1_beckman_routes.bp, name="root_beckman_routes")
         app.register_blueprint(v1_beckman_routes.bp, url_prefix="/v1")
+
+    if app.config.get("BECKMAN_ENABLE_V3", False):
+        from lighthouse.routes.v3 import beckman_v3_routes
+
+        app.register_blueprint(beckman_v3_routes.bp, name="root_beckman_routes")
+        app.register_blueprint(beckman_v3_routes.bp, url_prefix="/v1")
```

Comment:
Should these blueprints be consistently named with the ones above?
Also should the url_prefix be `/v3` for these?  Maybe not, I don't know the restrictions.